book/stdout_stderr_exit_codes.md

```diff
@@ -54,6 +54,24 @@ If we try to run the external `cat` on a file that doesn't exist, we can see wha
 ╰───────────┴─────────────────────────────────────────────╯
 ```
 
+## `echo`, `print`, and `log` commands
+
+The [`echo`](/commands/docs/echo.md) command is mainly for _pipes_. It returns its arguments, ignoring the piped-in value. There is usually little reason to use this over just writing the values as-is.
+
+In contrast, the [`print`](/commands/docs/print.md) command prints the given values to stdout as plain text. It can be used to write to standard error output, as well.
```

Comment:
maybe add that `print`ed things can not be piped?

for instance
```bash
[1,2,3] | each { |it|
    print $it
} | str join ", "
```
only prints
```bash
1
2
3
```
and not `1, 2, 3`